Libs/MRML/DisplayableManager/vtkMRMLAbstractDisplayableManager.cxx

```diff
@@ -690,7 +683,7 @@ vtkMRMLInteractionNode* vtkMRMLAbstractDisplayableManager::GetInteractionNode()
 //---------------------------------------------------------------------------
 vtkMRMLSelectionNode* vtkMRMLAbstractDisplayableManager::GetSelectionNode()
 {
-  return this->Internal->MRMLSelectionNode;
+  return vtkMRMLSelectionNode::SafeDownCast(this->GetMRMLScene()->GetNodeByID("vtkMRMLSelectionNodeSingleton"));
```

Comment:
I don't know if this makes sense for the selection node. The setting of active node is quite general and not specific to a particular view. May it could be associated with a layout instead ? 

Instead, I think we should move method like `vtkMRMLSelectionNode::AddNewPlaceNodeClassNameToList` to the interaction node. This will allow to customize the type of annotation that can be added in a specific views.

Reply:
Agreed, it seems to belong more to interaction than selection.